llvm/lib/MC/RepoObjectWriter.cpp

```diff
@@ -649,9 +650,13 @@ pstore::index::digest RepoObjectWriter::buildCompilationRecord(
   CompilationHash.update(Triple.size());
   CompilationHash.update(Triple);
 
+  // Only record all compilation memebers one time.
+  SmallSet<TicketNode *, 4> CMembers;
```

Comment:
What's the thinking behind the 4 here?

Reply:
Good question. I checked the SmallSet implementation. I think I should use DenseSet instead. In the SmallSet class, there is a static_assert of ‘N<32’. In the comment, it mentioned "In small mode SmallPtrSet uses linear search for the elements, so it is not a good idea to choose this value too high. You may consider using a DenseSet<> instead if you expect many elements in the set."

Changed SmallSet to DenseSet.